### ISSUE TITLE
[Request] Swipe Settings

### ISSUE THREAD

**Issue by Z3ROGRAV1TYx** [AnySoftKeyboard/AnySoftKeyboard#1774]
Would love to see these options for swipe:

- Auto space after swipe
- Pressing delete key after swipe will remove whole word
- Swipe color

**Comment by swedneck** [comment]
the first two things are pretty vital for swipe typing, without them i often find it faster to just type normally.

**Comment by Z3ROGRAV1TYx** [comment]
> 
> 
> the first two things are pretty vital for swipe typing, without them i often find it faster to just type normally.

Yeah, personally I also still find the word predictions terrible (Compared to Google's swipe).
For example if I swipe "Funny" it inputs "Finny". I mean how often does someone use the word "Finny"? And it's consistent too (At least). So If I swipe Funny multiple times, each time it inputs Finny. So I think it's really just down to detection and how it's dealing with suggestions.

**Comment by MarSoft** [comment]
Another issue: if I press Shift, and then swipe a word, I expect its first letter to be capitalized. But instead the word is typed in lowercase.
